docs/getting-started/setup.mdx

```diff
@@ -0,0 +1,174 @@
+# Setup
+
+The widgetbook package offers a Flutter Widget called `Widgetbook`, which enables you to inject custom widgets from your app. Before we start, let's take a look at a folder structure in Flutter projects.
+
+```dart
+your_flutter_project/
+├─ android/
+├─ ios/
+├─ lib/
+│ ├─ main.dart
+│ ├─ src/
+│ │ ├─ widgets/
+│ │ ├─ models/
+│ │ └─ services/
+│ └─ ...
+├─ assets/
+│ ├─ images/
+│ ├─ fonts/
+│ └─ ...
+├─ test/
+├─ pubspec.yaml
+└─ ...
+```
+
+To initialize Widgetbook in your Flutter project, follow the steps below:
+
+<Tabs
+  groupId="approach"
+  values={[
+    { label: 'Generator', value: 'generator' },
+    { label: 'Manual', value: 'manual' },
+  ]}
+>
+  <TabItem value="generator">
+
+1- Create a new file in your `lib` folder called `ui_catalogs.dart`. Alternatively, you can annotate an existing code element using the `@WidgetbookApp` annotation.
```

Comment:
I think the enumeration here has its problems as it's not easy to follow the "steps". The problem might be that we're using the `Tabs` which makes it harder to structure the documentation. How can we make it more readable?